rust/kernel/time.rs

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: GPL-2.0
+
+//! Timekeeping functions.
+//!
+//! C header: [`include/linux/ktime.h`](../../../../include/linux/ktime.h)
+//! C header: [`include/linux/timekeeping.h`](../../../../include/linux/timekeeping.h)
+
+use crate::bindings;
+use core::time::Duration;
```

Comment:
`hrtimer` wrapper is in my TODO list, so it's nice to see this ;-)

`core::time::Duration` has a slight problem of being more than 64bit, we probably need to introduce our own `Duration` as the counterpart of `ktime_t`. But it's just my own opinion, time subsystem maintainers may have different ones.

I don't think it blocks this change, just want to mention it while we are at it.